File: SE5_C#/EJ4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SE5_C_
{
    internal class EJ4
    {
        static void Main(string[] args)
        {
            int pares, impares, cero, cant, nums, cantnum;

            Console.Write("Ingrese la cantidad de números que desea tipear: ");
            cant = int.Parse(Console.ReadLine());
            cantnum = 0;
            pares = 0;
            impares = 0;
            cero = 0;
            do
            {
                Console.Write("Ingrese un número: ");
                nums = int.Parse(Console.ReadLine());
                cantnum++;

                if (nums % 2 == 0) 
                {
                    pares++;
                }
                if (nums % 2 != 0)
                {
                    impares++;
                }
                if (nums == 0) 
                {
                    cero++;
                }
            }
            while(cantnum<cant);

            Console.WriteLine($"La cantidad de números pares es: {pares}.");
            Console.WriteLine($"La cantidad de números impares es: {impares}.");
            Console.WriteLine($"La cantidad de ceros es: {cero}.");
        }
    }
}

File: S5P/EJ4.py
i=int
num=int
cceros=int
cpares=int
cimpares=int
cceros=0
cpares=0
cimpares=0

cant=int(input("Ingrese la cantidad de números: "))

for i in range(0,cant):
    num=int(input(f"Ingrese el número {i+1}: "))
    if num==0:
        cceros=cceros+1
    elif num%2==0:
        cpares+=1
    else:
        cimpares+=1

print(f"Cantidad de ceros: {cceros}")
print(f"Cantidad de pares: {cpares}")
print(f"Cantidad de impares: {cimpares}")







